### DISCUSSION TITLE
Busco equipo :D

### DISCUSSION THREAD

**Discussion by RobertoJimenez0626** [Launch-X-Latam/MisionFrontEnd#576] [Team Posts]
Hola, busco un equipo para el Hackaton.
Tengo experiencia en FrontEnd (HTML, CSS, y un poco de Javascript y React)
Si a alguien le falta un integrante este es mi correo [lx00261@innovaccion.mx](mailto:lx00261@innovaccion.mx)

**Comment by AndresGaliciaT** [comment]
yo también busco equipo lx16770@innovaccion.mx

**Comment by gapman20** [comment]
Yo también ando en busca de equipo si aun no tienen podríamos formar uno mi correo es lx16569@innovaccion.mx

**Comment by AntonioGB1** [comment]
Hola, también ando en busca de un equipo por si aun tienen lugar para sumarme. Mi correo es: lx04120@innovaccion.mx

**Comment by StephanyCS1** [comment]
Hola, busco equipo este es mi correo  ***@***.***

El mar, 22 nov 2022 a la(s) 20:04, Pedro Antonio Gregorio Bautista (
***@***.***) escribió:

> Hola, también ando en busca de un equipo por si aun tienen lugar para
> sumarme. Mi correo es: ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/114/comments/4>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ3KUS6ELV3JYB3KKTBKWHDWJVUTTANCNFSM6AAAAAASF3WHWA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Stephany F. Castro Salas*
Ingeniera Ambiental
Esp. Sistemas Integrados de Gestión QHSE
Esp. Gerencia De La Seguridad y Salud En El Trabajo
*3184002428*

*“Todos tus sueños pueden hacerse realidad si tienes el coraje de
perseguirlos”.Walt Disney*

Evita imprimir este mensaje si no es estrictamente necesario.

De esta manera ahorras agua, energía y recursos forestales

**Comment by StephanyCS1** [comment]
Hola, yo tambien estoy buscando equipo  este es mi correo
***@***.***

El lun, 21 nov 2022 a la(s) 21:33, Gabriel Alvarez ***@***.***)
escribió:

> Yo también ando en busca de equipo si aun no tienen podríamos formar uno
> mi correo es ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/114/comments/3>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ3KUSY3Z7GY4SVINWSVODTWJQWIJANCNFSM6AAAAAASF3WHWA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Stephany F. Castro Salas*
Ingeniera Ambiental
Esp. Sistemas Integrados de Gestión QHSE
Esp. Gerencia De La Seguridad y Salud En El Trabajo
*3184002428*

*“Todos tus sueños pueden hacerse realidad si tienes el coraje de
perseguirlos”.Walt Disney*

Evita imprimir este mensaje si no es estrictamente necesario.

De esta manera ahorras agua, energía y recursos forestales

**Comment by shelkanife** [comment]
¿Aún siguen buscando equipo?

**Comment by ozvec** [comment]
Seee todavía

**Comment by jlnunezobando** [comment]
Si, sigo buscando equipo.

**Comment by Talk1504** [comment]
Hola, yo estoy buscando equipo este es mi correo lx02169@innovaccion.mx

**Comment by IsmaelMorales16** [comment]
Buenas, yo busco equipo mi correo es : ***@***.***
________________________________
De: Ozvec ***@***.***>
Enviado: miércoles, 21 de diciembre de 2022 12:22
Cc: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Busco equipo :D (#114)


Seee todavía

Enviado desde Correo<https://go.microsoft.com/fwlink/?LinkId=550986> para Windows

De: ***@***.***>
Enviado: Wednesday, December 21, 2022 1:17 PM
CC: ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Busco equipo :D (#114)


¿Aún siguen buscando equipo?
